### PR TITLE
notifications-enhanced@hilyxx : first release

### PR DESCRIPTION
**Pull request by Hilyxx**
This applet is similar to system notification applet but with more options to manage easily notifications.

**Features:** 
- Add a new icon to panel when "Do not disturb" mod is enabled
- New menu label when "Do not disturb" mod is enabled
- Add a toggle switch to easily enable/disable notifications
- Add option to show or hide notification settings in menu
- Add an option to show "do not disturb" icon on panel when show empty tray option is disabled
- Tooltip now displays notification count

screenshot

(Sorry for re-upload)

### CODE REVIEW

**Comment by rcalixte** [comment]
You can run `./validate-spice notifications-enhanced@hilyxx` locally to save yourself some time as well.

File: notifications-enhanced@hilyxx/files/notifications-enhanced@hilyxx/po/notification-enhanced@hilyxx.pot [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+msgstr ""
+
+#: applet.js:418
+msgid "just now"
+msgstr ""
+
+#: applet.js:421
+msgid "%d second ago"
+msgid_plural "%d seconds ago"
+msgstr ""
```
Comment (rcalixte) [inline notifications-enhanced@hilyxx/files/notifications-enhanced@hilyxx/po/notification-enhanced@hilyxx.pot:57]:
```suggestion
msgstr[0] "%d second ago"
msgstr[1] "%d seconds ago"
```

File: notifications-enhanced@hilyxx/files/notifications-enhanced@hilyxx/po/notification-enhanced@hilyxx.pot [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+#: applet.js:425
+msgid "%d minute ago"
+msgid_plural "%d minutes ago"
+msgstr ""
```
Comment (rcalixte) [inline notifications-enhanced@hilyxx/files/notifications-enhanced@hilyxx/po/notification-enhanced@hilyxx.pot:62]:
```suggestion
msgstr[0] "%d minute ago"
msgstr[1] "%d minutes ago"
```

**Comment by rcalixte** [comment]
Running `./cinnamon-spices-makepot notifications-enhanced@hilyxx` will take care of any duplicate entries in the translations or the template.

**Comment by rcalixte** [comment]
I just pushed up the translation file changes. We should be all set to merge now.

**Review by rcalixte — APPROVED** [review]

**Comment by rcalixte** [comment]
It should be available online within a few minutes. Congrats!

**Comment by Hilyxx** [comment]
thank you @rcalixte .  .Po files are probably not for me...

**Comment by rcalixte** [comment]
> thank you @rcalixte . .Po files are probably not for me...

I always encourage to use the scripts that are provided. The goal is to make the lives of everyone involved easier. It also helps to standardize the files and remove the need for manual adjustments.

**Comment by yrtimiD** [comment]
Hi @Hilyxx , wanted to confirm, did you tried to inherit and extend original `notifications@cinnamon.org` applet instead of full clone to easier keep this fork up to date?

**Pull request merged**